requirements-cpu.txt

```
@@ -2,6 +2,6 @@
 -r requirements-common.txt
 
 # Dependencies for x86_64 CPUs
-torch == 2.3.1+cpu; platform_machine != "ppc64le"
+torch >= 2.3.1+cpu, <2.4.0+cpu; platform_machine != "ppc64le"
```

This looks broken:

You can test this with `env VLLM_CPU_DISABLE_AVX512=true VLLM_TARGET_DEVICE=cpu PIP_EXTRA_INDEX_URL=https://download.pytorch.org/whl/cpu pip install -e . -v`

I think it's enough to specify two torch lines:
```suggestion
torch >= 2.3.1+cpu; platform_machine != "ppc64le"
torch < 2.4.0+cpu; platform_machine != "ppc64le"
```